Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM node:18.18-alpine3.18
+# Keep this in sync with the Node version specified for the Conda dev environment (in environment_dev.yml)
+FROM node:22.8.0-alpine3.20
+
+RUN corepack enable
 
 WORKDIR /usr/src/app
 
 COPY . .
 RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn \
```

Comment:
```suggestion
# Use the --immutable flag as a precaution. This guarantees that the Docker
# build will fail if `yarn install` results in a different yarn.lock file than
# the one checked into the repo. This could happen for example if the version of
# Yarn used in one dev environment differs from the version used here.
RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn \
```